packages/wdio-cli/tests/utils.test.ts

```diff
@@ -398,6 +398,15 @@ describe('getCapabilities', () => {
     it('should return driver with capabilities for desktop', () => {
         expect(getCapabilities({ option: 'chrome' } as any)).toMatchSnapshot()
     })
+
+    it.only('should return driver with capabilities for wdio.config.js', () => {
+        jest.mock('@wdio/config', () => {
+            return {
+                configMock: jest.resetAllMocks()
```

Comment:
@christian-bromann  i tried everything , but still the entire methods from config file is returning mocked values. I am not sure how to get the actual values

Reply:
@christian-bromann 

https://github.com/webdriverio/webdriverio/blob/f51f23bae870f350aee8a71650fafc1eddfab5f0/packages/webdriver/tests/index.test.ts#L1

Adding unmok to first line of the test works. But it will affect other tests. nay tipson how to handle this

Reply:
No need to unmock this. What you want is to set a return value for the `getCapabilities` method. Or what are you trying to achieve?

Reply:
@christian-bromann  I want to cover below tests , 

1. When config file not exist it throws the error
2. When capability not provided , it throws error
3. When capability not found it throws error

Happy path

1. Parses capability array from config.js
2. Parses capbility object from config.ts

Reply:
@christian-bromann  sorted , could you have a look 😍